documents/Final Report/functionalities.tex

```diff
@@ -21,3 +27,9 @@ \section{Sell land functionality}
 
 \section{Request functionality}
 The agent is able to answer requests from other stakeholders. Requests are either accepted or declined. A request is declined when another agent wants to buy a piece of land that is planned for building a construction. Other requests that are beneficial for the agent’s indicators are accepted.
+
+\section{Buy land functionality}
+The agent can buy land. This will happen when the agent doesn't have any old buildings left to destroy, but it still wants to build new buildings. The agent will send a request to buy some land to another stakeholder, which will hopefully accept its offer. When it does accept the offer, the agent will build more buildings on this new piece of land.
```

Comment:
Instead of "The agent can buy land" you might want to consider "The agents is able to buy land"